tests/e2e/graph-node/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM graph-node:latest
+FROM graphprotocol/graph-node:latest
```

Comment:
@ryley-o you'll likely want to change this locally back to graph-node:latest so you can use your locally built graph-node image for M1

Reply:
ah, yes.
WDYT about maybe just using an env var and defining is simple dockerfile in-line in `docker-compose.yml`?

Reply:
(then could just define appropriate graph-node image in `test:e2e:ci` and `test:e2e` scripts)

Reply:
great idea! I added a separate script for m1 because I also pull the graphprotocol/graph-node image directly instead of rebuilding for m1 (m1 process didn't work for my m2 machine for some reason)